Source/ASButtonNode.mm

```diff
@@ -18,6 +21,22 @@
 #import <AsyncDisplayKit/ASImageNode.h>
 #import <AsyncDisplayKit/ASInternalHelpers.h>
 
+static void ASButtonNodeResolveHorizontalAlignmentForStyle(unowned ASLayoutElementStyle *style, ASStackLayoutDirection _direction, ASHorizontalAlignment _horizontalAlignment, ASStackLayoutJustifyContent _justifyContent, ASStackLayoutAlignItems _alignItems) {
+  if (_direction == ASStackLayoutDirectionHorizontal) {
+    style.justifyContent = justifyContent(_horizontalAlignment, _justifyContent);
+  } else {
+    style.alignItems = alignment(_horizontalAlignment, _alignItems);
+  }
+}
+
+static void ASButtonNodeResolveVerticalAlignmentForStyle(unowned ASLayoutElementStyle *style, ASStackLayoutDirection _direction, ASVerticalAlignment _verticalAlignment, ASStackLayoutJustifyContent _justifyContent, ASStackLayoutAlignItems _alignItems) {
```

Comment:
`unowned` in here has no effect. Parameters are implicitly owned by the caller so ARC doesn't emit retains/releases on them.